models/challenges.js

```diff
@@ -66,13 +67,13 @@ const fetchParticipantsData = async (participants) => {
 
 const postChallenge = async (challengeData) => {
   try {
-    const { start_date: startDate, end_date: endDate } = challengeData;
-    const startdate = new Firestore.Timestamp(startDate, 0);
-    const enddate = new Firestore.Timestamp(endDate, 0);
+    const { start_at: startAt, end_at: endAt } = challengeData;
+    const startat = new Firestore.Timestamp(getTimeInSecondAfter({ timestamp: startAt }), 0);
+    const endat = new Firestore.Timestamp(getTimeInSecondAfter({ timestamp: endAt }), 0);
     const challengeRef = await challengesModel.add({
       ...challengeData,
-      start_date: startdate,
-      end_date: enddate,
+      start_at: startat,
+      end_at: endat,
       participants: [],
       is_active: true,
     });
```

Comment:
I think it will be better to raise a different PR for changing the keys of challenges object because it might cause issue in frontend. What do you think?

Reply:
Yeah, you are right. Will do that.